cli/command/system/version.go

```diff
@@ -189,3 +211,46 @@ func getDetailsOrder(v types.ComponentVersion) []string {
 	sort.Strings(out)
 	return out
 }
+
+func getKubernetesVersion(dockerCli command.Cli, kubeConfig string) *kubernetesVersion {
+	if !dockerCli.ClientInfo().HasKubernetes() {
+		return nil
+	}
+
+	version := kubernetesVersion{
+		Kubernetes: "Unknown",
+		StackAPI:   "Unknown",
+	}
+	config, err := kubernetes.NewKubernetesConfig(kubeConfig)
+	if err != nil {
+		logrus.Debugf("failed to get Kubernetes configuration: %s", err)
+		return &version
+	}
+	version.StackAPI = getStackVersion(config)
+	version.Kubernetes = getKubernetesServerVersion(config)
+	return &version
+}
+
+func getStackVersion(kubeConfig *restclient.Config) string {
+	apiVersion, err := kubernetes.GetStackAPIVersion(kubeConfig)
+	if err != nil {
+		logrus.Debugf("failed to get Stack API version: %s", err)
+		return "Unknown"
+	}
+	return string(apiVersion)
+}
+
+func getKubernetesServerVersion(kubeConfig *restclient.Config) string {
+	version := "Unknown"
+	kubeClient, err := kubernetesClient.NewForConfig(kubeConfig)
```

Comment:
Could accept `kubeClient` instead of config, so the client can be passed to `GetStackAPIVersion` as well.

Reply:
Fixed